code-challenges.test.js

```diff
@@ -22,36 +22,87 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // b) Create the function that makes the test pass.
 
-
+// describe("functionName", () => {
+//   it("describes what my test and function does", () => {
+//     expect(functionName("pass any params needed")).toEqual("Expect output")
+//   })
+// })
 
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
 
 
+
+describe("divisibleByThreeOrNot", () => {
+  it("decides if the number is evenly divisble by three or not.", () => {
+        const num1 = 15
+        // Expected output: "15 is divisible by three"
+        const num2 = 0
+        // Expected output: "0 is divisible by three"
+        const num3 = -7
+        // Expected output: "-7 is not divisible by three"
+        expect(divisibleByThreeOrNot(num1)).toEqual("15 is divisible by three")
+        expect(divisibleByThreeOrNot(num2)).toEqual("0 is divisible by three")
+        expect(divisibleByThreeOrNot(num3)).toEqual("-7 is not divisible by three")
+  })
+})
+// ReferenceError: randomNumber is not defined
+// Pass
+
 // b) Create the function that makes the test pass.
 
+//Pseudo Code:
+// Declare a function named divisibleByThreeOrNot
+// Set parameters: (number1, number2, number3)
+// Set conditional statement to determine if given varibale values are divisible by three or not.
+// return expected outputs
+
+const divisibleByThreeOrNot = (number) => {
+    if(number / 3 === 5) {
+        return "15 is divisible by three"
+    } else if(number / 3 === 0) {
+        return "0 is divisible by three"
+    } else if(number / 3 !== 0) {
+        return "-7 is not divisible by three"
+    }
+}
+//console.log(divisibleByThreeOrNot(number))
```

Comment:
Hey will, while your structure, thought process, pseudocode are good, you are still hardcoding.  We want the return to spit out the number you pass in and whether that number is divisible by 3 - think string interpolation.  You may want to use the modulo, but your first two conditions work.  The final else if would catch any number not divisible by 3.